scripts/get-task-fuzzy-duration.py

```diff
@@ -97,14 +97,27 @@ def print_result(output_format, data):
 
 def investigate_task(args):
     time_before = time.time()
+    parent_task = get_json(
```

Comment:
What about changing a logic of that `while True` internals so we do not need two of this `get_json(...)` calls here?